### PR TITLE
[*] Permissions - Fix automated permission for projects having multiple teams

### PR DESCRIPTION
**Pull request by ArnaudValensi**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request
- [ ] Write changes made in the CHANGELOG.md
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by ArnaudValensi** [comment]
[(?) Permissions - Fix Liana permissions accross teams.](https://trello.com/c/5fVMaEEQ/2625-permissions-fix-liana-permissions-accross-teams)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Permissions - Fix permissions of the wrong team used for all others.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: Permissions - Fix automated permission for projects having multiple teams.

File: src/middlewares/permissions.js [hunk 2]
```diff
@@ -17,9 +19,10 @@ function createCheckPermission(environmentSecret, collectionName) {
 
   function checkPermissionListAndSearch(request, response, next) {
     const { searchToEdit } = request.query;
+    const renderingId = request.user.relationships.renderings.data[0].id;
```
Comment (arnaudbesnier) [inline src/middlewares/permissions.js]: Maybe you could create a centralized function to extract the `renderingId` from the user context.
Twice the same line of code in this file.

File: src/services/forest-server-requester.js [hunk 3]
```diff
@@ -5,30 +5,35 @@ const errorMessages = require('../utils/error-messages');
 const VError = require('verror');
 
 function ForestServerRequester() {
-  this.perform = (route, environmentSecret) => {
+  this.perform = (route, environmentSecret, queryParameters) => {
     const urlService = new ServiceUrlGetter().perform();
 
     return new P((resolve, reject) => {
-      request
+      const currentRequest = request
```
Comment (arnaudbesnier) [inline src/services/forest-server-requester.js]: I think you should do some renaming:
- `request` => `superagent`
- `currentRequest` => `request`

**Pull request merged**